data/endpoints/fps-initiate-payment-v3.json

```diff
@@ -1,777 +1,816 @@
+                      "schema": {
+                          "type": "string"
+                      }
                   }
-                },
-                {
-                  "name": "DigitalSignature",
-                  "in": "header",
-                  "description": "Signed hash of the body of the request. The hash is signed by your private key.",
-                  "required": true,
-                  "schema": {
-                    "type": "string"
+              ],
+              "requestBody": {
+                  "description": "Information that should be used to create a batch credit transfer.",
+                  "content": {
+                      "application/json": {
+                          "schema": {
+                              "$ref": "#/components/schemas/BatchCreateCreditTransferRequest"
+                          }
+                      }
+                  },
+                  "x-bodyName": "request"
+              },
+              "responses": {
+                  "202": {
+                      "description": "Accepted",
+                      "content": {
+                          "application/json": {
+                              "schema": {
+                                  "$ref": "#/components/schemas/BatchCreditTransferResponse"
+                              }
+                          }
+                      }
+                  },
+                  "400": {
+                      "description": "Bad Request",
+                      "content": {
+                          "application/json": {
+                              "schema": {
+                                  "$ref": "#/components/schemas/ValidationProblemDetails"
+                              }
+                          }
+                      }
+                  },
+                  "404": {
+                      "description": "Not Found"
+                  },
+                  "422": {
+                    "description": "Client Error",
+                    "content": {
+                      "application/json": {
+                        "schema": {
+                          "$ref": "#/components/schemas/ValidationProblemDetails"
+                        }
+                      }
+                    }
                   }
-                },
-                {
-                  "name": "X-Request-Id",
-                  "in": "header",
-                  "description": "A unique identifier for the request.",
-                  "required": true,
+              }
+          }
+      },
+      "/v3/payments/fps/return": {
+          "post": {
+            "tags": [
+              "FpsPayments"
+            ],
+            "summary": "Return a single FPS inbound payment. This endpoint cannot be used to return faster payments originating overseas.",
+            "description":"This endpoint is the recommended method for returning faster payments that are unable to be completed for any reason.",

```

Comment:
I'd have said "...faster payments that cannot be completed for any reason." Because it's not the payments themselves that are unable, as they are not autonomous agents.